apache/superset
# feat: Add dataset visuals for when a table is selected

### SUMMARY

This PR implements the visuals in the dataset panel that appear when a table is selected.

### ANIMATED GIF

![tableSelectedVisualsNoRefresh](https://user-images.githubusercontent.com/55605634/197089258-bff541ee-2326-473d-ac0b-235798519e71.gif)


### TESTING INSTRUCTIONS

- Go to `http://localhost:9000/dataset/add/?testing`
- Select a database, schema, and table
- Observe the table name header in the dataset panel with a table icon to the left, as well as a "Table columns" subheader below

### ADDITIONAL INFORMATION
